ckanext/spectrum/assets/css/spectrum.css

```diff
@@ -150,3 +167,96 @@ a.dropdown-item {
     padding: 10px;
     display: block;
 }
+
+
+/* Remove the separator in the dataset page */
+@media (min-width: 576px) {
+	.wrapper:before {
+		content: none;
+	}
+}
+
+/* Hide the navbar */
+@media (max-width: 768px) {
+	.navbar {
+		flex-wrap: nowrap;
+		justify-content: flex-start;
+	}
+}
+
+.btn-primary {
```

Comment:
I need to revert back to master branch to confirm that the button color is #49338f.